docs/content/docs/reference/react/protecting-routes.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Protecting routes'
+---
+
+## React Router
+
+When using [React Router](https://reactrouter.com/web/guides/quick-start) with `@nhost/react`, you can protect routes by creating an `AuthGate` component:
```

Comment:
"You can protect routes by creating an `AuthGate` component when using `@nhost/react` with [React Router](https://reactrouter.com/web/guides/quick-start)."